report_qweb_pdf_watermark/__manifest__.py

```diff
@@ -3,8 +3,10 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Pdf watermark",
-    "version": "10.0.1.0.0",
-    "author": "Therp BV,Odoo Community Association (OCA)",
+    "version": "10.0.2.0.0",
```

Comment:
This is a fix, not a new refactoring, so please change only last number. Check https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers